File: generate_accounts.kt
import kotlin.random.Random

fun generateAccountString(): String {
    val entries = mutableListOf<String>()
    
    for (i in 1..100) {
        val accountNumber = "account$i"
        val accountBalance = generateRandomBalance()
        entries.add("$accountNumber: $accountBalance")
    }
    
    return entries.joinToString(", ")
}

fun generateRandomBalance(): String {
    // Generate random balance between 100.00 and 90000.99
    val balance = Random.nextDouble(100.0, 90001.0)
    return String.format("%.2f", balance)
}

fun main() {
    val result = generateAccountString()
    println(result)
}

File: generate_accounts.py
import random

def generate_account_string():
    entries = []
    
    for i in range(1, 101):
        account_number = f"account{i}"
        account_balance = round(random.uniform(100.0, 90000.99), 2)
        entries.append(f"{account_number}: {account_balance:.2f}")
    
    return ", ".join(entries)

if __name__ == "__main__":
    result = generate_account_string()
    print(result)
